modules/sd-planning-considerations.adoc

```diff
@@ -70,10 +70,10 @@ The maximum number of compute nodes on ROSA is 180.
 
 .Rules for infrastructure node resizing alerts
 
-Resizing alerts are triggered for the infrastructure nodes in a cluster when either of the following scenarios are true: 
+Resizing alerts are triggered for the infrastructure nodes in a cluster when the CPU or the memory has high sustained utilization. This high-sustained utilization status is: 
```

Comment:
```suggestion
Resizing alerts are triggered for the infrastructure nodes in a cluster when it has high-sustained CPU or memory utilization. This high-sustained utilization status is: 
```

Standardizing the hyphenated `high-sustained` and removing the `the CPU`/`the memory` wording